### DISCUSSION TITLE
what is the text encoder used for?

### DISCUSSION THREAD

**Discussion by lucmian** [TheLastBen/fast-stable-diffusion#679] [General]
what is the text encoder used for? if I leave it disabled will it count as 0%¨?
and what is the difference if I leave 0% or 100%?

**Comment by TheLastBen** [comment]
The text_encoder connects text to image, 0% you won't be able to get the trained subject, 100% the trained subject will show everywhere

**Reply by lucmian** [reply]
if I leave it disabled it would be equal to what percentage?

**Reply by TheLastBen** [reply]
0%

**Reply by lucmian** [reply]
thanks
